1.0.0/mock/docker.md

```diff
@@ -0,0 +1,10 @@
+To run the mocks on Docker
```

Comment:
I would just change to "mock" in singular ;-)

Reply:
interesting, I believed plural because you have multiple mocks handled. But will survive switching to the singular.

As to the branching, would it not be easier to use the gitflow model? https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow